File: JavaApplication73/src/Boletin13_1/Adestrador.java
package Boletin13_1;

/**
 *
 * @author clamascabaleiro
 */
public class Adestrador extends Seleccion{
    private String idFederacion;

    public Adestrador() {
    }

    public Adestrador(int id, String nome, String apelido, int idade, String idFederacion) {
        super.setId(id);
        super.setNome(nome);
        super.setApelido(apelido);
        super.setIdade(idade);
        this.idFederacion = idFederacion;
    }

    public String getIdFederacion() {
        return idFederacion;
    }

    public void setIdFederacion(String idFederacion) {
        this.idFederacion = idFederacion;
    }
    
    @Override
    public void viaxar(){
        System.out.println("viaxan os adestradores");
    }
    
    public void dirixirPartido(){
       System.out.println("o adestrador " + super.getNome() + " dirixe o partido");
    }
    
    public void dirixirAdestramento(){
       System.out.println("o adestrador " + super.getNome() + " dirixe o entrenamento");
    }

    @Override
    public String toString() {
        return super.toString() + ", Adestrador{" + "idFederacion=" + idFederacion + '}';
    }
    
    
}

File: JavaApplication73/src/Boletin13_1/Adestrador.kt
package Boletin13_1

/**
 *
 * @author clamascabaleiro
 */
//Constructor primario y constructor de la superclase Seleccion
class Adestrador(var idFederacion: Int, id: Int, nome: String, apelido: String, edade: Int) : Seleccion (id, nome, apelido, edade) {


    @Override
    fun viaxar() {
        System.out.println("viaxan os adestradores")
    }

    fun dirixirPartido() {
        System.out.println("o adestrador " + super.getNome() + " dirixe o partido")
    }

    fun dirixirAdestramento() {
        System.out.println("o adestrador " + super.getNome() + " dirixe o entrenamento")
    }

    @Override
    fun toString(): String {
        return super.toString() + ", Adestrador{" + "idFederacion=" + idFederacion + '}'.toString()
    }

}

